04.class/memo.js

```diff
@@ -0,0 +1,109 @@
+const argv = require('minimist')(process.argv.slice(2))
+const fs = require('fs')
+const inquirer = require('inquirer')
+
+class MemoApp {
+
+  contructor () {
+    this.content = content
+  }
+
+  input () {
+    process.stdin.resume()
+    process.stdin.setEncoding('utf8')
+    let inputString = ''
+
+    process.stdin.on('data', function (chunk) {
+      inputString += chunk
+    })
+
+    process.stdin.on('end', function () {
+      const today = new Date
+
+      fs.writeFile(`${today}.txt`, inputString, (err) => {
+        if (err) console.log(err)
+        else console.log('保存しました')
+      })
+    })
+  }
+
+  list () {
+    fs.readdir('.', function (err, allFiles) {
+      if (err) throw err
+      const files = this.#filetypeConvert(allFiles)
+      files.forEach(function (file) {
+        const text = fs.readFileSync(file, 'utf8')
+        const lines = text.split(/\r\n|\n/)
+        console.log(lines[0])
+      })
+    })
+  }
+
+  reference () {
+    fs.readdir('.', function (err,allFiles) {
```

Comment:
eslintを使ってなかったら使うようにしてください〜。

```suggestion
    fs.readdir('.', function (err, allFiles) {
```

Reply:
あ、eslintは使ってるみたいですね。
こういうところは指摘してくれないのか・・・。

Reply:
最初の修正時にeslint実行するの忘れてしまっていました。
実行させていただきました。